src/schematics/cypress/schema.json

```diff
@@ -13,6 +13,15 @@
       },
       "x-prompt": "Would you like to remove Protractor from the project?"
     },
+    "addCypressTestScript": {
+      "description": " When true, Cypress test script will be added to package.json",
+      "type": "boolean",
+      "$default": {
```

Comment:
Please remove the `$default` and `x-prompt` properties here. This option should only be true if the user adds `--addCypressTestScripts` when adding the schematic.